packages/devextreme/js/__internal/grids/grid_core/views/m_rows_view.ts

```diff
@@ -792,8 +792,8 @@ export class RowsView extends ColumnsView {
     }, options));
 
     that._checkRowKeys(options.change);
-
     that._renderFreeSpaceRow($table, options.change);
+
     if (!that._hasHeight) {
```

Comment:
It's better to not change files, that have nothing to do with the fix. Let's remove these changes